src/network-client.ts

```diff
@@ -1,13 +1,14 @@
 import {
   GetAccountInfoRequest,
   GetAccountInfoResponse,
-} from '../generated/rpc/v1/account_info_pb'
-import { GetFeeRequest, GetFeeResponse } from '../generated/rpc/v1/fee_pb'
-import { GetTxRequest, GetTxResponse } from '../generated/rpc/v1/tx_pb'
+} from './generated/web/rpc/v1/account_info_pb'
```

Comment:
OOC: how come we can get away with using web types here but other places we need to union web types and node types (ex. `GetTxResponseWeb | GetTxResponseNode`)?

Reply:
Technically the output could be one of these two types but I've now set everything to fallback to web instead of being a union. Web types are a subset of the Node types.